src/extensions/default/InlineColorEditor/unittests.js

```diff
@@ -506,6 +506,53 @@ define(function (require, exports, module) {
                 
             });
             
+            describe("conversions in UPPER CASE", function () {
+                
+                beforeEach(function () {
+                    // Enable uppercase colors
+                    PreferencesManager.set("uppercaseColors", true);
+                });
+                afterEach(function () {
+                    // Re-disable uppercase colors
+                    PreferencesManager.set("uppercaseColors", false);
+                });
+                
+                it("should use uppercase colors", function () {
+                    expect(PreferencesManager.get("uppercaseColors")).toBe(true);
+                });
+                it("should convert a hex color to rgb when mode button clicked", function () {
```

Comment:
Each test should have a unique name, so you'll need to add something like "in uppercase" to each of these duplicated tests.